### DISCUSSION TITLE
How to upgrade the site? Are there relevant documents?

### DISCUSSION THREAD

**Discussion by pgy866** [OrchardCMS/OrchardCore#6983] [Q&A]
The layout style of my site is rewritten, and the data content remains unchanged. How can I upgrade my site? Is there any relevant documentation? Thank you

**Comment by Piedone** [comment]
Do you mean upgrading like upgrading to a new Orchard version? You can just update the NuGet packages to the new version and run the app. Most of the changes will be applied automatically. If there are breaking changes in the API then you'll get build errors, and if there are manual steps necessary between versions you'll see them in the release notes.

**Comment by pgy866** [comment]
Thank you, I haven't been to the community for a long time. Is there any related method for orchard core? https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Installing-and-upgrading-modules/

**Reply by Piedone** [reply]
Modules are commonly distributed as NuGet packages too so you can update them similarly. All in all, the update process with Orchard Core is much easier than with Orchard 1 and not unlike updating any other app that uses NuGet packages.

Is there anything specific you're interested in?

**Reply by pgy866** [reply]
Thank you very much, I rewritten the original website style and layout? The data remains unchanged, only the theme is upgraded and changed. I want to make multiple Themes for a site and switch at any time. Are there relevant documents? Maybe I need to look at the basic documentation?

**Reply by Piedone** [reply]
On theming, you can check out the docs for [the basics](https://docs.orchardcore.net/en/dev/docs/getting-started/theme/) and [info on templates](https://docs.orchardcore.net/en/dev/docs/reference/modules/Templates/), take a look at the built-in themes for inspiration, take part on Antoine's [upcoming workshop](https://www.orchardcore.net/workshops) or wait a bit for our Dojo Course on the topic.
